SMLHelper/FishFramework/ScaleFixer.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+
+namespace SMLHelper.V2.FishFramework
+{
+    public class ScaleFixer : MonoBehaviour
```

Comment:
This doesnt have to be a separate component and could be merged with the base one

Reply:
Ok i've renamed the component to be more generic, more functionality will probably be added later on